src/Uno.UI/UI/Xaml/Controls/TextBlock/TextBlock.wasm.cs

```diff
@@ -37,8 +50,45 @@ public TextBlock() : base("p")
 
 		}
 
+		private void ConditionalUpdate(ref bool condition, Action action)
+		{
+			if (condition)
+			{
+				condition = false;
+				action();
+			}
+		}
+
+		private void SynchronizeHtmlParagraphAttributes()
+		{
+			ConditionalUpdate(ref _fontStyleChanged, () => this.SetFontStyle(FontStyle));
+			ConditionalUpdate(ref _fontWeightChanged, () => this.SetFontWeight(FontWeight));
+			ConditionalUpdate(ref _fontFamilyChanged, () => this.SetFontFamily(FontFamily));
+			ConditionalUpdate(ref _fontSizeChanged, () => this.SetFontSize(FontSize));
+			ConditionalUpdate(ref _maxLinesChanged, () => this.SetMaxLines(MaxLines));
+			ConditionalUpdate(ref _textTrimmingChanged, () => this.SetTextTrimming(TextTrimming));
+			ConditionalUpdate(ref _textAlignmentChanged, () => this.SetTextAlignment(TextAlignment));
+			ConditionalUpdate(ref _lineHeightChanged, () => this.SetLineHeight(LineHeight));
+			ConditionalUpdate(ref _characterSpacingChanged, () => this.SetCharacterSpacing(CharacterSpacing));
+			ConditionalUpdate(ref _textDecorationsChanged, () => this.SetTextDecorations(TextDecorations));
+			ConditionalUpdate(ref _textWrappingChanged, () => this.SetTextWrapping(TextWrapping));
+			ConditionalUpdate(ref _paddingChangedChanged, () => this.SetTextPadding(Padding));
```

Comment:
We will have to do that in a single call ... in a future update :)

Reply:
https://github.com/unoplatform/uno/issues/2196